lib/music_library_controller.rb

```diff
@@ -0,0 +1,86 @@
+class MusicLibraryController
+  attr_accessor :path
+
+  def initialize(path = "./db/mp3s")
+    @path = path
+    MusicImporter.new(path).import
+  end
+
+  def call
+    input = ""
+
+    while input != "exit"
+    puts "Welcome to your music library!"
+    puts "To list all of your songs, enter 'list songs'."
+    puts "To list all of the artists in your library, enter 'list artists'."
+    puts "To list all of the genres in your library, enter 'list genres'."
+    puts "To list all of the songs by a particular artist, enter 'list artist'."
+    puts "To list all of the songs of a particular genre, enter 'list genre'."
+    puts "To play a song, enter 'play song'."
+    puts "To quit, type 'exit'."
+    puts "What would you like to do?"
+
+      input = gets.strip
+
+    list_songs if input == "list songs"
+    list_artists if input == "list artists"
+    list_genres if input == "list genres"
+    list_songs_by_artist if input == "list artist"
+    list_songs_by_genre if input == "list genre"
+    play_song if input == "play song"
+    end
```

Comment:
1. Fix the indentation here. It's hard to see what the `end` is for when it is misaligned.

2. Try moving some of your calculations out of `call`. Meaning, instead of a list of `puts` in `call`. `call` could invoke a new method, perhaps `welcome` (or whatever you want), and that method could store all the initial instructions. This would reduce the overall complexity of `call` and make it easier to work with for a future pass.